modules/internal/ChapelReduce.chpl

```diff
@@ -211,6 +211,32 @@ module ChapelReduce {
     proc clone() return new unmanaged MinReduceScanOp(eltType=eltType);
   }
 
+  class minmax: ReduceScanOp {
```

Comment:
Probably a question for @vasslitvinov -- it looks like min/max have parse time support where we map `min` to `MinReduceScanOp`. Do you know the motivation for that and do we need to do something similar here?

Reply:
I suspect that this is required for `min` and `max` to prevent them from resolving to the standalone `min()` and `max()` functions that are standard in Chapel.  As long as we don't have a standalone `minmax()` function, this shouldn't be a problem for minmax reductions and if/when we do, it probably won't be built-in, so a user will be able to use imports or use filters to select or avoid it as they wish.

Reply:
Good point. ChapelReduce.chpl certainly pollutes user space with internal-implementation symbols.

I forked off this question to #15796 .